blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/EnumMetric.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2013 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.types;
+
+import com.rackspacecloud.blueflood.exceptions.InvalidDataException;
+import com.rackspacecloud.blueflood.utils.TimeValue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+
```

Comment:
Note:  we might be able to refactor both this class and the Metric class to both extend from a BaseMetric class that implements IMetric, so that they can share some common code such as setTTL and getTTL.  

Let's decide whether we should do this now or later when we are actually implementing the writing the data to cassandra story.